runtime/src/main/java/org/corfudb/runtime/view/SealQuorumReplication.java

```diff
@@ -0,0 +1,57 @@
+package org.corfudb.runtime.view;
+
+import org.corfudb.runtime.CorfuRuntime;
+import org.corfudb.runtime.clients.BaseClient;
+import org.corfudb.runtime.exceptions.QuorumUnreachableException;
+
+import java.util.Arrays;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ExecutionException;
+
+/**
+ * Seals the Log unit servers in the Quorum Replication View
+ *
+ * Created by zlokhandwala on 3/7/17.
+ */
+public class SealQuorumReplication implements ISealReplicationView {
+
+    /**
+     * Seals the Layout Segment if quorum of log unit servers are sealed in every stripe.
+     * Else throws exception
+     *
+     * @param runtime       Runtime to set remote epoch for log unit servers.
+     * @param layoutSegment Layout segment to be sealed.
+     * @param sealEpoch     Epoch number to seal the servers with
+     * @return  True if seal successful
+     * @throws QuorumUnreachableException   Thrown if seal unsuccessful
+     */
+    @Override
+    public Boolean sealLayoutSegment(CorfuRuntime runtime, Layout.LayoutSegment layoutSegment, long sealEpoch) throws QuorumUnreachableException {
```

Comment:
Please add a backlog entry to seal the leftover log units. As of now there is no other process coming from behind to seal these leftover log units.